api/features/views.py

```diff
@@ -805,16 +806,16 @@ def get(self, request, identifier=None, *args, **kwargs):  # type: ignore[no-unt
         than a list.
         """
         if identifier:
-            return self._get_flags_response_with_identifier(request, identifier)  # type: ignore[no-untyped-call]
+            return self._get_flags_response_with_identifier(request, identifier)
 
         if "feature" in request.GET:
             feature_states = get_environment_flags_list(
                 environment=request.environment,
                 feature_name=request.GET["feature"],
                 additional_filters=self._additional_filters,
+                from_replica=True,
             )
-            if len(feature_states) != 1:
-                # TODO: what if more than one?
+            if not feature_states:
```

Comment:
I decided to ignore the `> 1` case here because, realistically, it makes no sense to have multiple features sharing the same exact name despite it curiously [not being unique](https://github.com/Flagsmith/flagsmith/blob/f980c352c729be160606a58b2bc816b49a3c1195/api/features/models.py#L98) per project. IMO this should be a concern of the data model, not of a view.